release/models/network-instance/openconfig-evpn.yang

```diff
@@ -1144,16 +1207,52 @@ module openconfig-evpn {
     }
   }
 
-  grouping evpn-endpoint-vni-state {
+  grouping evpn-endpoint-vni-config {
     description
-      "Grouping for L2VNI and L3VNI state information learned on the
-      local VXLAN Tunnel End Point from remote VTEPs";
+      "Grouping for L2VNI and L3VNI configuration parameters";
 
     leaf vni {
       type oc-evpn-types:evi-id;
       description "L2VNI or L3VNI Identifier";
     }
 
+    leaf vni-type {
+      type enumeration {
+        enum L2 {
+          description
+            "This is a Layer 2 service virtual network identifier (L2VNI)
+            that is used for communication within the same subnet or
+            broadcast domain";
+        }
+        enum L3 {
+          description
+            "This is a Layer 3 service virtual network identifier (L3VNI)
+            or VRF VNI that is used for communication between subnets";
+        }
+      }
+      description "The type of virtual network identfier";
+    }
```

Comment:
Is this redundant to the network-instance type?

Reply:
This is already part of the existing public model, it's just been refactored to pull it into a common location that can be used both by the existing `endpoint-vnis`, and the proposed `local-endpoint-vnis`

Reply:
got it